tests/e2e/workloads/pvc_snapshot_and_clone/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,153 @@
+    """
+    Tests to verify PVC status bound or not if
+    files size are smaller.
+    """
+
+    @bugzilla("2039265")
+    def test_pvc_status_after_clone(self, pvc_factory, teardown_factory):
+        """
+        1. Create 50Mib PVC
+        2. Attach PVC to an application pod
+        3. Download Linux kernel and untar it to the directory where PVC is mounted
+        4. Create a new PVC from the snapshot
+        5. Take a snapshot of the PVC.
+        6. Create a new PVC out of that snapshot.
+        7. Verify PVC state
+
+        Args:
+            pvc_factory: A fixture to create new pvc
+            teardown_factory: A fixture to destroy objects
+        """
+        pvc_size = 90
+        worker_nodes_list = get_worker_nodes()
+        kernel_url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.4.tar.xz"
+        download_path = "tmp"
+        dir_path = os.path.join(os.getcwd(), download_path)
+        file_path = os.path.join(dir_path, "linux-4.4.tar.xz")
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+        urllib.request.urlretrieve(kernel_url, file_path)
+
+        # Create a RWX PVC
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            access_mode=constants.ACCESS_MODE_RWX,
+            status=constants.STATUS_BOUND,
+            size=pvc_size,
+            size_unit="Mi",
+        )
+
+        # Confirm PVCs are Bound
+        log.info("Verifying the CephFS PVC are Bound")
+        helpers.wait_for_resource_state(
+            resource=pvc_obj, state=constants.STATUS_BOUND, timeout=240
+        )
+        pvc_obj.reload()
+        log.info("Verified: CephFS PVC are Bound")
+
+        # Create a pod on a particular node
+        selected_node = random.choice(worker_nodes_list)
+        log.info(f"Creating a pod on node: {selected_node} with pvc {pvc_obj.name}")
+
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHFILESYSTEM,
+            pvc_name=pvc_obj.name,
+            namespace=pvc_obj.namespace,
+            node_name=selected_node,
+            pod_dict_path=constants.PERF_POD_YAML,
+        )
+        teardown_factory(pod_obj)
+
+        # Confirm that the pod is running on the selected_node
+        helpers.wait_for_resource_state(
+            resource=pod_obj, state=constants.STATUS_RUNNING, timeout=180
+        )
+        pod_obj.reload()
+        assert pod.verify_node_name(
+            pod_obj, selected_node
+        ), "Pod is running on a different node than the selected node"
+
+        pod_name = pod_obj.name
+        pod_path = "/mnt"
+        _ocp = OCP(namespace=pvc_obj.namespace)
+
+        rsh_cmd = f"cp {dir_path} {pod_name}:{pod_path}"
+        _ocp.exec_oc_cmd(rsh_cmd)
+
+        rsh_cmd = f"exec {pod_name} -- tar xvf {pod_path}/tmp/linux-4.4.tar.xz -C {pod_path}/tmp"
+        try:
+            _ocp.exec_oc_cmd(rsh_cmd)
+        except CommandFailed:
+            log.warning("Command failed")
```

Comment:
Its better to fill the PVC rather than skipping on command failure.. Check on the size of the PVC that needs to be created. https://bugzilla.redhat.com/show_bug.cgi?id=2039265#c7